src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -476,9 +476,9 @@ static s32 D_808535E4 = 0;
 static f32 D_808535E8 = 1.0f;
 static f32 D_808535EC = 1.0f;
 static u32 D_808535F0 = 0;
-static u32 D_808535F4 = 0;
-static s16 D_808535F8 = 0;
-static s16 D_808535FC = 0;
+static u32 sConveyorSpeedIdx = 0;
+static s16 sIsFloorConveyor = false;
```

Comment:
would do Index personally. we talked about changing it everywhere and that can happen in a later pass, but might as well not introduce more